src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -554,10 +570,10 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 	}
 }
 
-function preventDefaultAndExecCommand(e: KeyboardEvent, commandId: string) {
+function preventDefaultAndExecCommand(e: KeyboardEvent, commandId: string, showUI?: boolean, value?: string) {
 	// use preventDefault() to avoid invoking the editor's select all
 	e.preventDefault();
```

Comment:
When I do ctrl + b, the text bolds but the left side Views also toggles. I think adding `e.stopPropagation()` here will fix that.